assets/style.css

```diff
@@ -50,16 +50,18 @@ body{
 
 .card h1 {
     font-weight: 700;
+    font-family: 'Outfit', sans-serif;
     color: var(--Dark-blue);
-    font-size: 1.2rem;
+    font-size: 15px;
     padding: .2rem 1rem 0rem;
-    margin-top: 10px;
+    margin-top: 12px;
 }
 
 .card p{
     font-weight: 400;
     font-size: .78rem;
     padding: 0rem .7rem .5rem;
-    color: var(--Grayish-blue);
+    color: hsl(220, 15%, 55);
```

Comment:
The color is actually hsl(220, 15%, 55%). Switch back to using the variable but update the variable to have the correct color.